integration_tests/steps/integration_tests.py

```diff
@@ -320,7 +320,8 @@ def test_offer_bid_files(context):
                  os.path.join(base_path, 'grid', 'house-2-balancing-offers.csv')]
 
     assert all(len(glob.glob(f)) == 1 for f in file_list)
-    assert all(len(open(glob.glob(f)[0]).readlines()) > 1 for f in file_list)
+    for f in file_list:
```

Comment:
It will not be possible to see the filename like that. You need to add a string to the assert to display the exact filename that failed

Reply:
Wow! Great, did not know the functionality of assert (adding an error string) 
Thanks for that!